### PR TITLE
Provide a custom format for condor_ce_status and startd ads.

### PR DESCRIPTION
**Pull request by bbockelm**
This is analogous to what we have for scheduler ads already.  Sample output:

```
$ condor_ce_status -pool red.unl.edu | head
Worker Node          State     Payload ID      User       Scheduler                         Job Runtime BatchID    BatchUser  Jobs   Pilot Age 
red-c1806.unl.edu    Claimed   145633.39       cmsdataops cmssrv218.fnal.gov                0+05:25:30  5430917.0  cmsprod       9   0+20:36:51
red-c1806.unl.edu    Claimed   92962.67        cmst1      vocms0311.cern.ch                 0+02:36:45  5431503.0  cmsprod       2   0+17:57:01
red-c1806.unl.edu    Claimed   93015.16        cmst1      vocms0311.cern.ch                 0+10:09:06  5432124.0  cmsprod       1   0+10:15:20
red-c1806.unl.edu    Claimed   538265.88       cmst1      vocms0309.cern.ch                 0+00:49:26  5431813.0  cmsprod       4   0+12:41:15
red-c1806.unl.edu    Claimed   538095.2        cmst1      vocms0309.cern.ch                 0+04:46:18  5433050.0  cmsprod       2   0+05:20:06
red-c1806.unl.edu    Claimed   538258.36       cmst1      vocms0309.cern.ch                 0+01:04:22  5431913.0  cmsprod       3   0+12:25:28
red-c1806.unl.edu    Claimed   145771.98       cmsdataops cmssrv218.fnal.gov                0+05:18:25  5430399.0  cmsprod       8   1+04:38:24
red-c1807.unl.edu    Claimed   537995.44       cmst1      vocms0309.cern.ch                 0+03:37:07  5431731.0  cmsprod       3   0+14:40:29
red-c1807.unl.edu    Unclaimed                                                              0+00:14:30  5434985.0  nanohub       0   0+00:14:59
```

### CODE REVIEW

File: config/01-common-collector-defaults.conf [hunk 1]
```diff
@@ -11,4 +11,5 @@
 COLLECTOR_PORT=9619
 
 STATUS_DEFAULT_SCHEDD_PRINT_FORMAT_FILE=/usr/share/condor-ce/ce-status.cpf
+STATUS_DEFAULT_STARTD_PRINT_FORMAT_FILE=/usr/share/condor-ce/pilot-status.cpf
```
Comment (brianhlin) [inline config/01-common-collector-defaults.conf]: Shouldn't this change also go into `config/01-common-collector.conf`?

Comment (bbockelm) [inline config/01-common-collector-defaults.conf]: Unless there's a particular reason the admin may want to set it, I'd prefer not.

(FWIW - I consider the /etc version of everything somewhat of a mistake and want to keep as much as reasonable in /usr/share)

**Comment by brianhlin** [comment]
Looks good to me, could you fix the merge conflicts?

File: config/pilot-status.cpf [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
+SELECT
+  Machine AS "Worker Node" WIDTH -20
+  State AS State WIDTH -9
+  ifThenElse(State=="Claimed", split(GlobalJobId, "#")[1], "") AS "Payload ID" WIDTH -15
+  ifThenElse(State=="Claimed", split(RemoteUser, "@")[0], "") AS User WIDTH -10
+  ifThenElse(State=="Claimed", split(GlobalJobId, "#")[0], "") AS "Scheduler" WIDTH -31
+  ifThenElse(EnteredCurrentActivity isnt undefined, EnteredCurrentActivity, time()) AS "  Job Runtime" PRINTAS ACTIVITY_TIME
+  GLIDEIN_SiteWMS_JobId AS BatchID WIDTH -10
+  #strcat(GLIDEIN_ClusterId) AS "Factory Job" WIDTH -11
```
Comment (brianhlin) [inline config/pilot-status.cpf]: Also, any reason you're keeping the commented lines in here?

**Comment by bbockelm** [comment]
@brianhlin or @djw8605 - I'm a bit swamped.  Could either of you poke at addressing the issues @brianhlin points out above?

File: config/pilot-status.cpf [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+SELECT
+  Machine AS "Worker Node" WIDTH -20
+  State AS State WIDTH -9
+  ifThenElse(State=="Claimed", split(GlobalJobId, "#")[1], "") AS "Payload ID" WIDTH -15
+  ifThenElse(State=="Claimed", split(RemoteUser, "@")[0], "") AS User WIDTH -10
+  ifThenElse(State=="Claimed", split(GlobalJobId, "#")[0], "") AS "Scheduler" WIDTH -31
+  ifThenElse(EnteredCurrentActivity isnt undefined, EnteredCurrentActivity, time()) AS "  Job Runtime" PRINTAS ACTIVITY_TIME
+  GLIDEIN_SiteWMS_JobId AS BatchID WIDTH -10
```
Comment (djw8605) [inline config/pilot-status.cpf]: It actually shows both.  The first GlobalJobId is the job id of the payload job, ie the job id on the remote submission host.  The jobid on the local CE's batch system (Slurm for example) is shown in the BatchID column.

**Comment by djw8605** [comment]
@brianhlin Updated pull request from my testing.  Only issue was 248ff30, in which the pilot-status.cpf was not installed inside the spec file.

**Comment by brianhlin** [comment]
Looks good to me.

**Pull request merged**